### ISSUE TITLE
Check mqttx 1.5.4 release for macos

### ISSUE THREAD

**Issue by alexpts** [emqx/MQTTX#569]
I download mqttx 1.5.4, and after install i see version 1.5.3. Possible release is bad.

mqttx: SHA256 mismatch (https://github.com/emqx/MQTTX/releases/download/v1.5.4/MQTTX-1.5.4-mac.zip)
Expected: e6aa8692b77f5aa99b961f0f35836d01ca7728b29ae3c6bbfb157ed1ab73830a
Actual: b4c6b9c8dba5f7b78c5cd25298352e45c0fc4dd9d4cca878f7e581cc5a7150b6

Please check release artifact 1.5.4 and update release.

**Comment by ysfscream** [comment]
YES. Where did you download it? I refreshed the release packages yesterday, there should be no such problem on GitHub releases.

**Comment by alexpts** [comment]
I downloaded manual from github at 9 may.
I try download dmg now. It updated correct.

I try update via homebrew now. 
Formula in homebrew has not work now. Brew has old link to 1.5.3 instead 1.5.4.

**Comment by alexpts** [comment]
I think about 1.5.4.1

**Comment by alexpts** [comment]
Why 1.5.4.1 is invalid?
https://semver.org/


Given a version number MAJOR.MINOR.PATCH, increment the:

_MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes._

**Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.**

**Comment by ysfscream** [comment]
Thank you for sharing, but 1.5.4.1 does not conform to the agreement set by our version, so some patch updates are required to make the patch version, and then release 1.5.5.

MAJOR.MINOR.PATCH
